system/Debug/Toolbar/Views/toolbar.css

```diff
@@ -1,8 +1,8 @@
 /*! CodeIgniter 4 - Debug bar
```

Comment:
Should the header be copied from the php source files?

Reply:
And in addition, should all scss files have one?

Reply:
I think yes, for consistency.